packages/libs/react-ui/src/components/Badge/Badge.stories.tsx

```diff
@@ -25,7 +25,14 @@ const meta: Meta<IBadgeProps> = {
     },
     style: {
       control: { type: 'radio' },
-      options: ['default', 'inverse'],
+      options: [
+        'default',
+        'inverse',
+        'info',
+        'warning',
+        'positive',
+        'negative',
+      ],
```

Comment:
I made something for this! you can use the getVariants function.

```typescript
import { getVariants } from '../../storyDecorators/getVariants';

const {style} = getVariants(badge);
```

An example is in the Button.stories.tsx file.